packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -787,6 +788,32 @@ class MediaQuery extends InheritedWidget {
     );
   }
 
+  /// Creates a [_MediaQueryFromWindow] which is builds and updates a
+  /// [MediaQuery] using the latest [WidgetsBinding.window] values.
+  ///
+  /// The [MediaQuery] is wrapped in a separate widget to ensure that only it
+  /// and its dependents are updated when `window` changes instead of rebuilding
```

Comment:
Nit:
```suggestion
  /// and its dependents are updated when `window` changes. instead of rebuilding
```

Reply:
Did you mean a comma here? The dot didn't make sense for me 🤔
I added a comma for now, let me know

Reply:
Ah yes, thank you. Pointed out a typo with a typo. 😋